README.md

```diff
@@ -35,6 +35,16 @@ pip install awsiotsdk
 pip install ./aws-iot-device-sdk-python-v2
 ```
 
+## Install for Raspberry Pi ([buster](https://www.raspberrypi.org/downloads/raspbian/))
```

Comment:
Let's change the language here, since these steps aren't even necessary if you're using python 3 on Raspberry Pi (most users). These extra steps useful for **any** platform/python-ver combo where a prebuilt wheel isn't accessible (ex: other esoteric IoT devices)

How about:
## Installation Issues
`awsiotsdk` depends on [awscrt](https://github.com/awslabs/aws-crt-python), which makes use of C extensions. Precompiled wheels are downloaded when installing on major platforms (Mac, Windows, Linux, Raspbian (python3 only)). If wheels are unavailable for your platform (ex: Raspbian with python2.7), your machine must compile some C libraries. If you encounter issues, install the following and try again:
```
sudo apt-get install cmake
sudo apt-get install libssl-dev
```

Reply:
Looks great. I'll fix my pull request.